src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        Ball randomBall = new Ball(ColorSupplier.getRandomColor(), randomNumber);
```

Comment:
you should better define fields for `Random` and `ColorSupplier` in order not to create new instances of those every time this method is called

```
private final Random = new Random();
private final ColorSupplier = new ColorSupplier();
```